Index: ItemListPanel.java
===================================================================
--- ItemListPanel.java	(revision 1349)
+++ ItemListPanel.java	(working copy)
@@ -18,6 +18,7 @@
 
 import info.jtrac.domain.AbstractItem;
 import info.jtrac.domain.ColumnHeading;
+import info.jtrac.domain.Item;
 import info.jtrac.domain.ColumnHeading.Name;
 import info.jtrac.domain.History;
 import info.jtrac.domain.ItemSearch;
@@ -318,7 +319,8 @@
                                     value = new PropertyModel(item, "assignedTo.name");
                                     break;
                                 case TIME_STAMP:
-                                    value = new Model(DateUtils.formatTimeStamp(item.getTimeStamp()));
+                                    //value = new Model(DateUtils.formatTimeStamp(item.getTimeStamp()));
+                                    value = new Model(DateUtils.formatTimeStamp(((Item)item).getLatestHistory().getTimeStamp()));
                                     break;
                                 case SPACE:
                                     if(showHistory) {
